src/Akka.Streams.Kafka/Settings/ConsumerSettings.cs

```diff
@@ -119,14 +120,24 @@ public sealed class ConsumerSettings<TKey, TValue>
         /// </summary>
         public string DispatcherId { get; }
         /// <summary>
+        /// Allow automatic topic creation on the broker when subscribing to or assigning a topic.
+        /// </summary>
+        /// <remarks>
+        /// See more here: https://kafka.apache.org/documentation/#allow.auto.create.topics
+        /// Additionally, due to https://github.com/confluentinc/confluent-kafka-dotnet/issues/1366 ,
+        /// when set to `true` and topic is not created by Confluent driver, consuming error will be ignored
+        /// (like if no message to consume)
+        /// </remarks>
+        public bool AutoCreateTopicsEnabled { get; }
```

Comment:
Need to add our own issue about this change to the repo, but we can do that in a subsequent PR

Reply:
Added an issue: https://github.com/akkadotnet/Akka.Streams.Kafka/issues/158